src/platform/utilities/ui/focus.js

```diff
@@ -66,9 +95,12 @@ export function focusElement(selectorOrElement, options, root) {
  *  component we're waiting for (could be an element in shadow DOM)
  * @example waitForRenderThenFocus('h3', document.querySelector('va-radio').shadowRoot);
  */
-const isCypressRunning = typeof Cypress !== 'undefined' && Cypress.env('CI');
+const isCypressRunning =
+  typeof Cypress !== 'undefined' &&
+  (Cypress.env('CI') || environment.isLocalhost());
```

Comment:
I don't think `environment.isLocalhost()` means Cypress is running?
I would add an additional `isLocalHost = environment.isLocalhost();` constant, then adjust the statements below to do something like `const defaultTime = isCypressRunning || isLocalhost ? 0 : 250;`

Reply:
We only want to reduce the defaultTime when Cypress is running locally or in CI. If I only use `isLocalHost` then when I manually test the pages locally (outside of Cypress), this function may not focus on a header inside a web component. Having the focus on the header doesn't matter as much when Cypress is running tests, especially keyboard-only tests, at full speed.

Reply:
In that case I suggest we rename `isCypressRunning` to something more declarative to what it's doing?

Reply:
Done, renamed to `noAsyncFocusWhenCypressRunningInCiOrLocally`

Reply:
I also removed the code setting the `maxIteration` to `1` since that change doesn't matter anymore